android/src/main/java/com/reactnativecommunity/netinfo/NetworkCallbackConnectivityReceiver.java

```diff
@@ -104,6 +104,10 @@ void updateAndSend() {
                             NetworkCapabilities.NET_CAPABILITY_VALIDATED)
                             && !isInternetSuspended;
 
+            if (isInternetReachable) {
+                connection.bindProcessToNetwork(null);
```

Comment:
Can't you use `getConnectivityManager()` here too and remove the new `public ConnectivityManager connection;`?

Reply:
Yes sure! Thank you for your suggestion :D